windows/src/engine/keyman32/K32_load.cpp

```diff
@@ -917,26 +899,21 @@ BOOL VerifyKeyboard(LPBYTE filebase, DWORD sz)
   if(ckbp->dwFileVersion < VERSION_MIN ||
      ckbp->dwFileVersion > VERSION_MAX)
   {
-    /* Old or new version -- identify the desired program version */
-    if(VerifyChecksum(filebase, sz))
-    {
-      for(csp = (PCOMP_STORE)(filebase + ckbp->dpStoreArray), i = 0; i < ckbp->cxStoreArray; i++, csp++)
-        if(csp->dwSystemID == TSS_COMPILEDVERSION)
-        {
-          char buf2[256];
-          if(csp->dpString == 0)
-            wsprintf(buf2, "errWrongFileVersion:NULL");
-          else
-            wsprintf(buf2, "errWrongFileVersion:%10.10ls", StringOffset(filebase, csp->dpString));
-          Err(buf2);
-          return FALSE;
-        }
+    for(csp = (PCOMP_STORE)(filebase + ckbp->dpStoreArray), i = 0; i < ckbp->cxStoreArray; i++, csp++) {
```

Comment:
:+1: on adding `{` to match code style guide. Slightly squirmy that the closing brace is actually transparently included from the deleted `if() {` in this patch, but it is correct afaict. 😆